src/distribution.cpp

```diff
@@ -381,8 +362,6 @@ UPtrDist distribution_from_xml(pugi::xml_node node)
     dist = UPtrDist {new Watt(node)};
   } else if (type == "normal") {
     dist = UPtrDist {new Normal(node)};
-  } else if (type == "muir") {
```

Comment:
We've added support for older XML files with "muir" nodes for the Python API, but not here on the C++ side. I'm wondering if some kind of similar backward support is warranted. Or perhaps an error message asking the user to update their XML files with the Python API? 

As far as I can tell, the presence of a Muir distribution in the `settings.xml` is going to produce the error message ` ERROR: Invalid distribution type: muir`, which may be confusing. Glancing this over quickly, so my apologies if I'm missing something.

Reply:
Good point. I left in support for `from_xml` on the Python side so that users with older files have a way to regenerate them (`from_xml`, `export_to_xml`). Keeping support on the C++ side would require more code, hence why I removed it. I'll add an error message here as you suggest telling users to update their files.